zebra/zebra_srv6_vty.c

```diff
@@ -298,7 +305,7 @@ DEFPY (locator_prefix,
 	 *      user should use a pattern of zeros as a filler.
 	 *  (3) The Node Id portion (LSBs) cannot exceed 24 bits.
 	 */
-	locator->block_bits_length = prefix->prefixlen - 24;
+	locator->block_bits_length = 40;
```

Comment:
BGP_PREFIX_SID_SRV6_LOCATOR_BLOCK_LENGTH?

Reply:
Actually, 40 is the same value as `BGP_PREFIX_SID_SRV6_LOCATOR_BLOCK_LENGTH`. However, it's defined for bgpd. so, I'll define `ZEBRA_SRV6_LOCATOR_BLOCK_LENGTH` and `ZEBRA_SRV6_LOCATOR_NODE_LENGTH` and replace magic numbers with new defined constants.